plugins/org.eclipse.reddeer.swt/src/org/eclipse/reddeer/swt/api/Label.java

```diff
@@ -10,8 +10,8 @@
  *******************************************************************************/
 package org.eclipse.reddeer.swt.api;
 
-
 import org.eclipse.swt.graphics.Image;
+
```

Comment:
That's why I asked you for code formatting in other classes (we will avoid such changes in the future).